zuul_alpha/core.py

```diff
@@ -85,8 +84,19 @@ def _get_plaintext_private_key(
 
         return response[u'Plaintext']
 
+    class Encryptor:
```

Comment:
Yea. 

Can you elaborate on that, though? I'd like to know why it's an anti-pattern here or what issues it might cause.